src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -401,11 +402,37 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
 
         if ("student".equals(view) || responses.isEmpty()) {
             return "";
+        } else {
+            Map<String, Integer> answerFrequency = new LinkedHashMap<String, Integer>();
+            String numChoice = getQuestionResultStatistics(responses, answerFrequency);
+            if ("".equals(numChoice)) {
+                return "";
+            } else {
+                DecimalFormat df = new DecimalFormat("#.##");
+                StringBuilder fragments = new StringBuilder();
+                for (Entry<String, Integer> entry : answerFrequency.entrySet()) {
+                    fragments.append(Templates.populateTemplate(
+                            FormTemplates.MCQ_RESULT_STATS_OPTIONFRAGMENT,
+                            Slots.MCQ_CHOICE_VALUE, entry.getKey(),
+                            Slots.COUNT, entry.getValue().toString(),
+                            Slots.PERCENTAGE,
+                            df.format(100 * checkForNaN(Integer.parseInt(numChoice), (double) entry.getValue()))));
+
+                }
+                // Use same template as MCQ for now, until they need to be
+                // different.
+                return Templates.populateTemplate(FormTemplates.MCQ_RESULT_STATS, Slots.FRAGMENTS,
+                        fragments.toString());
+            }
         }
+    }
 
-        boolean isContainsNonEmptyResponse = false; // we will only show stats if there is at least one nonempty response
+    private String getQuestionResultStatistics(List<FeedbackResponseAttributes> responses,
```

Comment:
If i return int i can't differentiate between `""` and `0` in the getQuestionResultStatisticsCsv() and getQuestionResultStatisticsHtml() methods.